web/app_dev.php

```diff
@@ -13,8 +13,9 @@
     || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
     || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1')) || php_sapi_name() === 'cli-server')
 ) {
-    header('HTTP/1.0 403 Forbidden');
-    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
+//    header('HTTP/1.0 403 Forbidden');
```

Comment:
Fixed!

--
Mikael Öhrén

2016-04-15 14:30 GMT+02:00 Jon Gotlin <notifications@github.com>:

> In web/app_dev.php
> <https://github.com/kompisbyran/Kompisbyran/pull/86#discussion_r59866544>:
>
> > @@ -13,8 +13,9 @@
> >      || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
> >      || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1')) || php_sapi_name() === 'cli-server')
> >  ) {
> > -    header('HTTP/1.0 403 Forbidden');
> > -    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
> > +//    header('HTTP/1.0 403 Forbidden');
>
> Fix this please.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kompisbyran/Kompisbyran/pull/86/files/1047dd4e8ab8714d3c3691cbadf6425b7fec2a1e#r59866544>
>